.github/workflows/deploy_mavis.yml

```
@@ -76,7 +76,7 @@ jobs:
       - name: Upload artifact
         uses: actions/upload-artifact@v4
         with:
-          name: tfplan
+          name: tfplan_mavis
```

```suggestion
          name: tfplan_app
```

Could we call one task deploying the infrastructure and one task deploying the app? I think that'll help to clarify what these jobs do as Mavis, in my mind at least, is the whole service, including the infrastructure and the app.